FirebaseRemoteConfig/Sources/FIRRemoteConfigComponent.m

```diff
@@ -107,4 +115,15 @@ + (void)load {
   return @[ rcProvider ];
 }
 
+- (void)rolloutsStateDidUpdate:(nonnull FIRRolloutsState *)rolloutsState {
+  // calling the subscriber to update the change
+  [self.subscribers[@"nameSpace"] onRolloutsStateChanged:rolloutsState];
```

Comment:
I think you'll need to refer to the variable nameSpace here, or get the current namespace somehow. We might be able to get away with just using the default app's namespace

Reply:
Yup, will change this for actual implementation!